stable/jenkins/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: jenkins
 home: https://jenkins.io/
-version: 1.15.0
+version: 1.15.1
```

Comment:
As it's a new feature 1.16.0 would be fine.

Reply:
Fixed.